guides/source/error_reporting.md

```diff
@@ -88,7 +88,7 @@ If no error is raised in the block, `Rails.error.handle` will return the result
 
 ```ruby
 user = Rails.error.handle(fallback: -> { User.anonymous }) do
-  User.find_by(params[:id])
+  User.find(params[:id])
```

Comment:
`find` will raise if the record isn't found.
```suggestion
  User.find_by(id: params[:id])
```

Reply:
@p8 `find_by` will not raise an exception so if I understand correctly, fallback will not be applied.

Reply:
Ah, my bad. 👍